spec/suggestions_spec.rb

```diff
@@ -1,40 +1,4 @@
+# frozen_string_literal: true
+
 require_relative 'spec_helper'
 require_relative '../lib/nsidc_open_search/dataset/search/solr_search_suggest'
```

Comment:
Looks like all tests were removed. Can this file be deleted? Was the logic moved somewhere else?

Reply:
Let me take a look.  The change to this file was an automated one from rubocop, so I overlooked that the file itself was empty.